app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
+  queue_as :wakes_metrics
```

Comment:
That's a good point regarding how idle our 'metrics_worker` queue is . Depending upon what you think about moving the job to the app vs keeping it in the gem, I am fine with making separate queues and then running them on the same dynos. 

I was thinking about a separate queue mostly to avoid any collisions (I imagine that during the hour when we update all facebook metrics the worker will be quite busy), and use queue level operations that Sidekiq admin provides.